01-introduction.Rmd

```diff
@@ -113,121 +167,140 @@ Because R is open source, these developments can easily build on previous work,
 This approach is advantageous because it encourages collaboration and avoids 'reinventing the wheel'.
 The package **sf** (covered in Chapter \@ref(spatial-class)), for example, builds on its predecessor **sp**.
 
-Shifts in R's spatial ecosystem have been influenced by shifts in the wider R community, as exemplified by the visualization and data processing packages **ggplot2** and **dplyr** (released in [2007](https://cran.r-project.org/src/contrib/Archive/ggplot2/) and [2014](https://cran.r-project.org/src/contrib/Archive/dplyr/) respectively).
```

Comment:
I absolutely agree with you, that the tidyverse has had a large impact on the R community (at the UseR I had the impression that every second talk had to do with the tidyverse...). 
But I would suggest to stick to dplyr here, and not mention ggplot2 that prominently, that's all. We talk about ggplot2 later on (see below).